### DISCUSSION TITLE
images not being displayed after export to gitHub

### DISCUSSION THREAD

**Discussion by MarcVdH** [cables-gl/cables_docs#490] [! Welcome - readme first!]
Hi all, 

I created a patch with 132 small (34kb) jpg`s
I use a TextureArrayLoader to show them.
The patch runs well in Cables, but when I export to Github I get an error message. 
"EXC ERROR: could not find file : nXXX.jpg?rnd=b471....
Nevertheless, when I check the assets folder in Github, the images are there. but unfortunately they are not shown when the patch is running.

I work on windows 10. The images are not used anywhere else and uploaded into the patch fresh.
I hope someone can help me to fix the issue?

Thanks in advance,
MarcV

![missing images upon export](https://user-images.githubusercontent.com/126583906/221914901-43728992-132a-4bde-8a68-52138a4a11f3.jpg)
![missing images upon export II](https://user-images.githubusercontent.com/126583906/221914926-fd5e3e2e-2fc6-4473-a90e-5b3543d47506.jpg)

**Comment by steam0r** [comment]
would be helpful to get access to the patch, or a similar one, know your export settings, see any errors on the dev-console or network, and maybe even access your github-pages and repository. this way it seeems like, yes it does not find the images :)

**Comment by MarcVdH** [comment]
links to the patch and Github page...


https://github.com/MarcVdH/timeboat

https://cables.gl/p/63cc1a6dc77d7fd5823be40a?s=sFb1HmsOwbwQ6tS7

**Comment by steam0r** [comment]
so your deployment is not working because you are loading images from `sandbox.cables.gl` (which is not allowed, as you can see by the CORS errors in console):

<img width="805" alt="Bildschirm­foto 2023-02-28 um 17 46 57" src="https://user-images.githubusercontent.com/151271/221921507-6b4e247a-16bf-44b1-8705-1e87036bacb8.png">

this is because you set this in your assetloader:

<img width="837" alt="Bildschirm­foto 2023-02-28 um 17 49 10" src="https://user-images.githubusercontent.com/151271/221921599-86336f55-04b3-4800-8fbc-60a1aebf29ec.png">

change this to a relative url and/or use https://cables.gl/op/Ops.Cables.AssetPathURL

also:

https://cables.gl/docs/4_export_embed/cors/cors

**Comment by MarcVdH** [comment]
sweet, thanks for the quick reply,
my main issue is that i do not even know even the basics of all the "technical stuff".
I`ll try to make sense of it all and make it work later on... 

Thanks again !
